test_client/tests/src/com/owncloud/android/lib/test_project/test/GetShareesTest.java

```diff
@@ -190,6 +191,59 @@ public void testGetRemoteShareesOperation() {
 		assertTrue(!result.isSuccess() && result.getHttpCode() == HttpStatus.SC_BAD_REQUEST);
 	}
 	
+	/**
+	 *  Test get federated sharees
+	 * 
+	 *  Requires OC server 8.2 or later
+	 */
+	public void testGetFederatedShareesOperation() {
+		Log.v(LOG_TAG, "testGetFederatedSharees in");
+
+		/// successful cases
+
+		// search for sharees including "@"
+		GetRemoteShareesOperation getShareesOperation = new GetRemoteShareesOperation("@", 1, 50);
+		RemoteOperationResult result = getShareesOperation.execute(mClient);
+		JSONObject resultItem;
+		JSONObject value;
+		int type;
+		int fedCount = 0;
+		assertTrue(result.isSuccess() && result.getData().size() > 0);
+		try {
+			for (int i=0; i<result.getData().size(); i++) {
```

Comment:
Weird indentation here again; take a look to the options in your editor, please.